.circleci/config.yml

```diff
@@ -39,14 +41,16 @@ jobs:
     executor: python_executor
     steps:
       - checkout
+      - run: git submodule sync
+      - run: git submodule update --init
       - run:
           name: Install dependencies
           command: |
             curl -sfL https://install.goreleaser.com/github.com/ValeLint/vale.sh | sh -s v1.7.1
       - run:
           name: Run Vale
           command: |
-            ./bin/vale --config .circleci/vale/.vale.ini --glob='*.{md}' --output=line . | tee vale.out
+            ./bin/vale --config common/build_tools/vale/.vale.ini --glob='*.{md}' --output=line . | tee vale.out
```

Comment:
Just double checking this path is right? /vale/ or ./vale for the config we supply?